lgc/builder/BuilderImpl.cpp

```diff
@@ -503,36 +521,130 @@ static Value *traceNonUniformIndex(Value *nonUniformVal) {
 
   return candidateIndex;
 }
+#endif
```

Comment:
Add a comment on the condition of the #if

Reply:
Done.